src/main/java/me/modmuss50/optifabric/mod/OptifineSetup.java

```diff
@@ -226,40 +218,15 @@ private static Path getMinecraftJar() {
 			}
 		}
 
-		Path minecraftJar = getLaunchMinecraftJar();
```

Comment:
`getLaunchMinecraftJar` works in dev, so I just removed all this as it's being unhelpful